### DISCUSSION TITLE
PCL2主页下载缓存在哪里

### DISCUSSION THREAD

**Discussion by SqrtTan** [Meloong-Git/PCL#4073] [提问] [answered]
我想看看信标的主页文件咋写的，但是不知道缓存在哪……

**Comment by MoYuan-CN** [comment, marked as answer]
> 不知道缓存在哪……

`C:\Users\{username}\AppData\Local\Temp\PCL\Cache\Custom.xaml`

---

> 我想看看信标的主页文件咋写的

https://github.com/Light-Beacon/PCL2-NewsHomepage
